client/app/components/add-threads/addThreads.controller.js

```diff
@@ -0,0 +1,35 @@
+angular.module('appThreadAdditionController', [])
+  .controller('threadAdditionController', ['$scope', '$timeout', 'threadAdditionService', 'utils',
+  function($scope, $timeout, threadAdditionService, utils) {
+      $scope.init = function() {
+        }
+
```

Comment:
drobnostka, ale tabulacja się rozjechała i to pod spodem wygląda jakby należało do init()

Reply:
done